api/GPUSupportedFeatures.json

```
@@ -189,6 +189,45 @@
           }
         }
       },
+      "feature_core-features-and-limits": {
+        "__compat": {
+          "description": "`core-features-and-limits` feature",
+          "spec_url": "https://gpuweb.github.io/gpuweb/#core-features-and-limits",
+          "tags": [
+            "web-features:webgpu"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "139",
+              "notes": "Currently available on all adapters and enabled automatically on all devices even if not requested."
+            },
+            "chrome_android": "mirror",
+            "deno": {
+              "version_added": false
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
```

Firefox also advertises this feature, based on https://bugzilla.mozilla.org/show_bug.cgi?id=1958278, 

```suggestion
          "firefox": {
            "version_added": "141",
            "partial_implementation": true,
            "notes": "Currently supported on Windows only, in all contexts except for service workers."
            },
```